1906-11-24.xml

```diff
@@ -1794,21 +1810,23 @@
                             lost interest, but the vitality that held as ebbed and flowed with her
                             absence or presence. Mme. Florian and M. Garat as the de Morenes were
                             certainly excellent, characteristic, amusing, if a trifle forced at
-                            moments. Lucienne (Mile. Mac-Leane) was very charming. M Aroaud again
+                            moments. Lucienne (Mile. Mac-Leane) was very charming. M. Arnaud again
                             was the thorough homme da monde" as we have already seen him. I except
                             from the ' rived this morning by the Hamburg-American "loss of vitality"
                             charge his talk with Gaston liner Oceans, de River on dueling. He had
                             some delightful things to say, and he said them admirably, giving the
                             exact value to every point: "Je n'ai, poar ainsi dire, jamais ete tnd
-                            !"— most clever of many olever sayings ! He marked them all without over
+                            !"— most clever of many clever sayings ! He marked them all without over
                             emphasizing any. It is certainly a play to see and to read more than
-                            once for its quiet wit an 1 truth, and (can one use the phrase ?)
-                            perfect breeding I apologize for the word, but English is not the best
-                            of tongues for criticism of drama. One has to take what one can get.</p>
+                            once for its quiet wit and truth, and (can one use the phrase ?) perfect
+                            breeding I apologize for the word, but English is not the best of
+                            tongues for criticism of drama. One has to take what one can get.</p>
                     </div>
                     <div type="item">
                         <head>La Chaelaine</head>
                         <p>Sapho</p>
+                    </div>
+                    <div type="item" status="verified">
```

Comment:
don't create a new item separate from its headline--add the status attribute to the div tag at line 1825